docs/get-started/overview.md

```diff
@@ -0,0 +1,237 @@
+# Get Started
+
+Getting started to experience the power of core futures of GreptimeDB.
+
+## Install GreptimeDB
+You can try out GreptimeDB with our test builds released on [GitHub](https://github.com/GreptimeTeam/greptimedb/releases) and [Dockerhub](https://hub.docker.com/r/greptime/greptimedb). Note that GreptimeDB is currently under intense development. So these binaries are **not ready to be used in the production environment**.
+
+### Linux & Mac
+
+For Linux and macOS users, you can download the latest build of the `greptime` binary by using the following commands:
+
+```shell
+curl -L https://raw.githubusercontent.com/GreptimeTeam/greptimedb/develop/scripts/install.sh | sh
+```
+
+When the download is complete, the binary file `greptime` will be stored in your current directory. 
+
+You can run the GreptimeDB in standalone mode:
+
+```shell
+./greptime standalone start
+```
+
+For a complete specification of configuration options of GreptimeDB, see the [configuration documentation](/guides/operations/configuration.md).
+
+### Docker
+Make sure the [Docker](https://www.docker.com/) is already installed. If not, you can follow the official [documents](https://www.docker.com/get-started/) to install Docker.
+
+```shell
+docker run -p 4000-4004:4000-4004 \
+-p 4242:4242 -v "$(pwd)/greptimedb:/tmp/greptimedb" \
+--name greptime --rm \
+greptime/greptimedb standalone start \
+--http-addr 0.0.0.0:4000 \
+--rpc-addr 0.0.0.0:4001 \
+--mysql-addr 0.0.0.0:4002 \
+--postgres-addr 0.0.0.0:4003 \
+--prom-addr 0.0.0.0:4004 \
+--opentsdb-addr 0.0.0.0:4242
+```
+
+The data will be stored in the `greptimedb/` directory in your current directory. For a complete specification of configuration options of GreptimeDB, see the [configuration documentation](./../guides/operations/configuration.md).
```

Comment:
For a complete specification of configuration options of GreptimeDB, see the [configuration documentation](./../guides/operations/configuration.md).

Reply:
This part is mentioned before, does it have to be repeated?

Reply:
No, I will remove one of them and extract it to the top `Install GreptimeDB` chapter.